definitions/tools/picard_merge_vcfs.cwl

```diff
@@ -3,10 +3,10 @@
 cwlVersion: v1.0
 class: CommandLineTool
 label: "Picard MergeVcfs"
-baseCommand: ["/usr/bin/java", "-jar", "/opt/picard/picard.jar", "MergeVcfs"]
+baseCommand: ["/usr/bin/java", "-jar", "-Xmx38g", "/opt/picard/picard.jar", "MergeVcfs"]
```

Comment:
I realized there's a bug here--the memory param got stuck between `-jar` and its value.  There is a fix for that as part of #960.